components/Cards/examples/CardsCollectionTintingExample.m

```diff
@@ -75,6 +83,22 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
                                                 forIndexPath:indexPath];
   [cell setTintColor:[UIColor redColor]];
   [cell setNeedsDisplay];
+
+  // enable interactable mode and accessibility for interactivity.
+  if (_interactable) {
+    cell.interactable = YES;
+    cell.selectable = YES;
+
+    cell.isAccessibilityElement = YES;
+    cell.accessibilityCustomActions = @[
+      [[UIAccessibilityCustomAction alloc] initWithName:@"Toggle Switch"
+          target:cell selector:@selector(customAccessibilityActionToggleSwitch)],
```

Comment:
Nit: Colons must be aligned, [different lines cannot have different numbers of arguments on them](http://google.github.io/styleguide/objcguide.html#spacing-and-formatting).